src/pudl/etl/epacems_assets.py

```diff
@@ -44,23 +44,33 @@ def hourly_emissions_epacems(
     epacems_settings = context.resources.dataset_settings.epacems
 
     schema = Resource.from_id("hourly_emissions_epacems").to_pyarrow()
-    epacems_path = (
+    partitioned_path = (
         Path(context.op_config["pudl_output_path"]) / "hourly_emissions_epacems"
     )
-    epacems_path.mkdir(exist_ok=True)
+    partitioned_path.mkdir(exist_ok=True)
+    monolithic_path = (
+        Path(context.op_config["pudl_output_path"]) / "epacems_output.parquet"
```

Comment:
Up until now we have been writing the monolithic output to `hourly_emissions_epacems.parquet` -- is there a reason to change the filename? This will break things.

Reply:
No reason, fixed now